src/EnergyPlus/Autosizing/HeatingAirflowUASizing.cc

```diff
@@ -54,57 +54,9 @@
 
 namespace EnergyPlus {
 
-    void HeatingAirflowUASizer::initializeForAPI(EnergyPlusData &EP_UNUSED(state),
```

Comment:
I took out all the initializeForAPI stuff for now.  That can be added back later and just gets in the way of the core changes here.